packages/pilot/src/containers/Reprocess/Form/index.js

```diff
@@ -1,103 +1,187 @@
+                  holderName: t('holder_name'),
+                  transactionId: t('transaction'),
+                }}
+              />
+            </Col>
+          </Row>
+          {allowReprocessWithoutAntifraud
+            && (
+              <Row flex>
+                <Col className={styles.alertColumn}>
+                  <DescriptionAlert
+                    content={(
+                      <span>
+                        {t('pages.reprocess.without_antifraud_disclaimer_1')}
+                        <b>{t('pages.reprocess.without_antifraud_disclaimer_2')}</b>
+                        {t('pages.reprocess.without_antifraud_disclaimer_3')}
+                        <a
+                          target="_blank"
+                          rel="noopener noreferrer"
+                          href={t('pages.reprocess.without_antifraud_documentation_link')}
+                        >
+                          {t('pages.reprocess.without_antifraud_disclaimer_4')}
+                        </a>
+                      </span>
+                    )}
+                    icon={<WarningIcon />}
+                    title={t('attention')}
+                    type="error"
+                  />
+                </Col>
+              </Row>
+            )
+          }
+        </Grid>
+      </ModalContent>
+      <div className={styles.actions}>
         <ModalActions>
+          {allowReprocessWithoutAntifraud
+            && (
+              <Tooltip
+                content={t(`pages.reprocess.without_antifraud_requirements.${lockReason}`)}
+                placement="topStart"
+              >
+                <Button
+                  disabled={disableWithoutAntifraudReprocess}
+                  fill="outline"
+                  onClick={onReprocessWithoutAntifraud}
+                >
+                  {t('pages.reprocess.reprocess_without_antifraud')}
+                </Button>
+              </Tooltip>
+            )
+          }
           <Button
             disabled={loading}
             fill="gradient"
-            type="submit"
+            onClick={onReprocessWithAntifraud}
           >
-            {t('pages.reprocess.submit')}
+            {allowReprocessWithoutAntifraud
+              ? t('pages.reprocess.reprocess_with_antifraud')
+              : t('pages.reprocess.reprocess')
+            }
           </Button>
         </ModalActions>
-      </form>
-    )
-  }
+      </div>
+    </Fragment>
+  )
 }
 
 ReprocessForm.propTypes = {
+  allowReprocessWithoutAntifraud: PropTypes.bool.isRequired,
   amount: PropTypes.number.isRequired,
-  cardFirstDigits: PropTypes.string.isRequired,
-  cardLastDigits: PropTypes.string.isRequired,
+  disableWithoutAntifraudReprocess: PropTypes.bool,
   holderName: PropTypes.string.isRequired,
-  installments: PropTypes.number.isRequired,
   loading: PropTypes.bool,
-  onConfirm: PropTypes.func.isRequired,
+  lockReason: (props, propName, componentName) => {
```

Comment:
Só para manter um pouco mais fácil a leitura, esta função poderia ser nomeada e declarada fora do objeto `PropTypes`
```suggestion
  const validateLockReason = (props, propName, componentName) => {
  ...
  lockReason: validateLockReason
```

Reply:
Feito!